src/gsi/setupt.f90

```diff
@@ -1284,7 +1284,7 @@ subroutine init_vars_
      varname='ps'
      call gsi_bundlegetpointer(gsi_metguess_bundle(1),trim(varname),rank2,istatus)
      if (istatus==0) then
-         if(allocated(ges_z))then
```

Comment:
With the removal of ges_z on lines 1287, 1378, and 1396, this variable is no longer used and is being flagged while compiling in debug mode.  Please remove ges_z on line 325.